packages/functional-tests/tests/subscription-tests/support.spec.ts

```diff
@@ -37,45 +37,17 @@ test.describe('severity-1 #smoke', () => {
 
 test.describe('severity-2 #smoke', () => {
   test.describe('support form with active subscriptions', () => {
-    test.beforeEach(() => {
-      test.slow();
-    });
-
-    test('go to support form, submits the form', async ({
-      pages: { login, relier, subscribe, settings, subscriptionManagement },
-    }, { project }) => {
-      test.skip(
-        project.name === 'production',
-        'no real payment method available in prod'
-      );
-      await relier.goto();
-      await relier.clickSubscribe();
-      await subscribe.setConfirmPaymentCheckbox();
-      await subscribe.setFullName();
-      await subscribe.setCreditCardInfo();
-      await subscribe.clickPayNow();
-      await subscribe.submit();
-
-      //Login to FxA account
-      await login.goto();
-      await login.clickSignIn();
-      const subscriptionPage = await settings.clickPaidSubscriptions();
-      subscriptionManagement.page = subscriptionPage;
-      await subscriptionManagement.fillSupportForm();
-
-      //Since we don't have proper Zendesk config in CircleCI, the form cannot be successfully submitted
```

Comment:
Need to see if we can find the Zendesk ticket in Jira for who can help get this unstuck in staging.

Reply:
I followed up on this in today's SubPlat weekly meeting (see [notes](https://docs.google.com/document/d/14U8gCJUOD8QaXDAvHY01e2quu0eDrVpajDBU5pAnQbs/edit)).  We were unable to find the ticket. Solving this issue is not currently on the roadmap and conflicts with plans to re-architect the support ticket (Zendesk) code. For now documenting the limitation in code is considered sufficient.

Reply:
Sure sure... So if the plan is that this is dead, +1 to deleting code.
If we think it will be back sooner rather than later do we want to just change this to using a more generic `test.skip()` so we're shamed by having _n_ skipped tests that we'll eventually need to rectify.

Also, if we're unable to find the Jira ticket (i tried looking during our morning 1:1:1 but failed finding anything obvs for Zendesk) I say we refile the ticket so we can track it as a work item and smash that like and subscribe.

Reply:
I think the issue is that we have a lot of tests that haven't been revisited or documented in full as to why they are failing, possible resolutions, when and how to resolve, etc. By the time we do get to them, like this audit, we are unable to go back and find these necessary details or related tickets.

Since this is an issue that isn't on the roadmap and thus will not be resolved in the near future, removing the code altogether and documenting this decision is a good approach.

If we are confident in how we can resolve this with little effort, then I would opt for a `test.fixme` as Kat has done in other tests, but in this case, since there is no viable solution right now, I am aligned with removing this code.